di_website/workforus/blocks.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+from wagtail.core.blocks import (
+    PageChooserBlock,
+    RichTextBlock,
+    StreamBlock,
+    StructBlock,
+    TextBlock,
+    URLBlock
+)
+from wagtail.images.blocks import ImageChooserBlock
+
+from di_website.common.constants import RICHTEXT_FEATURES
+from di_website.common.blocks import LinkBlock
+
+
+class BenefitsBlock(StructBlock):
+    title = TextBlock()
+    body = RichTextBlock(required=False)
+    image = ImageChooserBlock(required=False)
+
+    class Meta():
+        icon = 'fa-heart'
+
+
+class BenefitsStreamBlock(StreamBlock):
+    item = BenefitsBlock()
+    required = False
+
+
+class TeamStoryBlock(StructBlock):
+    title = TextBlock()
+    story_url = URLBlock()
+    image = ImageChooserBlock(required=False)
+
+    class Meta():
+        icon = 'fa-book'
+
+
+class TeamStoryStreamBlock(StreamBlock):
+    item = TeamStoryBlock()
+    required = False
```

Comment:
New line at the end of the file... You use VSCode so I suggest you install the `EditorConfig` extension so you won't have to worry about these. It'll add them automatically on save, as is configured in the `.editorconfig`

Reply:
Updated newline and installed EditorConfig extension for VSCode